### DISCUSSION TITLE
How to migrate from Prisma to Drizzle?

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#90] [Q&A]
Hi,

Personally I preffer [Drizzle](https://orm.drizzle.team/) over Prisma as Drizzle is a query builder which aims to be close to SQL syntax.
Is there a quick way to migrate from Prisma to Drizzle?

**Comment by kentcdodds** [comment]
I don't plan on including (in the template or docs) ways to migrate away from the built-in decisions. But if you'd like to do so then you're welcome to do so :)

**Reply by thefrana** [reply]
@kentcdodds Have you thought about moving from Prisma to Drizzle in the official Epic Stack? I have not found a corresponding decision. Why have you decided to use Prisma? In my eyes, Drizzle is much closer to pure SQL which I think aligns with the philosophy of Epic Stack. I would like to know your ideas on this if I may.

**Reply by kentcdodds** [reply]
I've not written why I prefer Prisma over Drizzle yet, but I suppose I probably should eventually. I have given this some thought and I do feel like Prisma aligns better with my own sensibilities and needs.

**Reply by thefrana** [reply]
Thank you! I am sure it would be interesting to read your thoughts on this.

**Reply by hinsxd** [reply]
I think drizzle is fairly easy to use in any projects. Unlike prisma which runs `prisma generate` to put everything in node_modules (which might not be the usual way), drizzle is like any other ORM that requires you to create a client with credentials then export it.

**Reply by jacobparis** [reply]
If it helps I have a little guide for setting up Drizzle in remix, and then you can just go through the prisma calls and replace them one by one

https://www.jacobparis.com/content/remix-drizzle

I like prisma for the Epic Stack because its API is basically pseudocode. You don't need to know prisma to be able to read the query/mutation and know what it's doing. Anyone who knows any other ORM or DB language can read it and reimplement their preferred choice, but the same can't be said of the inverse.

**Reply by stephen776** [reply]
A year later - I do wonder if this would be worth considering again. drizzle has matured a lot, the new query api allows for better syntax when querying, and the community seems to be growing.

**Reply by kentcdodds** [reply]
I'm not personally interested. The new typed SQL feature of prisma solved my one major gripe.

**Reply by stephen776** [reply]
Thanks Kent. I'll have to try out the typed sql feature. 


Are there any other Prisma features that tip the scale for you?

**Reply by kentcdodds** [reply]
I like its API, schema config, and especially the migrations.
